digits/dataset/images/generic/test_lmdb_creator.py

```diff
@@ -26,9 +26,9 @@
 import caffe_pb2
 
 
-IMAGE_SIZE  = 10
+IMAGE_SIZE  = 8
 TRAIN_IMAGE_COUNT = 100
-VAL_IMAGE_COUNT = 20
+VAL_IMAGE_COUNT = 10
```

Comment:
on my machine the two changes from this file don't seem to produce a measurable speed up. Do we need them?